### ISSUE TITLE
Improve discoverability of translations

### ISSUE THREAD

**Issue by harding** [bitcoinops/bitcoinops.github.io#595]
When trying to reply to [this comment](https://www.reddit.com/r/Bitcoin/comments/o144wf/bitcoin_optech_newsletter_153/h20orry/) on Reddit, it took me a while to find links to the Japanese translations---and I knew they were there.  I don't have any specific suggestions, but I think we could certainly improve discoverability.

Related: #285 for a Japanese RSS feed.

**Comment by Xekyo** [comment]
Yeah, I definitely see what you mean. Maybe we could put the [en/ja/es] in the head navigation, if it doesn't look too obtrusive?

**Comment by bitschmidty** [comment]
It seems the lack of translation was not clear in the reddit post since the person was viewing an individual newsletter page that did not (yet) have a translation. One way to indicate the (potential, future) availability of translation could be to show a gray out link for languages that are not yet available. As opposed to hiding the translation language bar entirely.

Here is an example idea:
<img width="897" alt="Screen Shot 2021-06-29 at 3 09 39 PM" src="https://user-images.githubusercontent.com/1615772/123861034-4f584b00-d8ec-11eb-8540-85c55ddbfb89.png">

**Comment by Xekyo** [comment]
Hah, I didn't know that this was there either, because I usually only glance at it right after we publish it. :sweat_smile:

We only have a couple Spanish translations overall, but the Japanese ones come out regularly. It might look a bit silly if we always had the greyed out Spanish there (although it might get some people to notice and pick it up!).

**Comment by jnewbery** [comment]
The greyed out link for languages that don't yet have a translation could direct to a page that says something like "The {language} translation for this newsletter is not available yet. If you'd like to help translate or review, please contact the optech team/go to the github/etc. In the mean time, here are the previous newsletters that have been translated into {language}".
